android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetActivity.java

```diff
@@ -217,6 +216,11 @@ protected void onParticipantLeft(HashMap<String, Object> extraData) {
         }
     }
 
+    protected void onReadyToClose() {
+        JitsiMeetLogger.i("SDK is ready to close");
+        finish();
+    }
+
```

Comment:
Isn't it necessary to override this in MainActivity, similar to what we had in onConferenceTerminated?

Reply:
No, because the welcome page is enabled and thus we'll never render the blank page, which triggers the event.